dev/bender/plugins/ckeditor5/static/extensions.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals bender, CKEDITOR, describe: true */
+
+'use strict';
+
+// ### General extensions.
+( function() {
+	// Make Bender wait to start running tests.
+	var done = bender.defer();
+
+	// Wait for the "ckeditor" module to be ready to start testing.
+	CKEDITOR.require( [ 'ckeditor' ], done );
+} )();
+
+// ### AMD related extensions.
+( function() {
+	var wasRequireCalled,
+		requires;
+
+	/**
+	 * AMD tools related to CKEditor.
+	 */
+	bender.amd = {
+		/**
+		 * Specifies the list of CKEditor modules to be loaded before tests start. The modules will be passed to the
+		 * describe() functions as parameters.
+		 *
+		 * @params {...String} module The name of the module to load.
+		 */
+		require: function() {
+			if ( wasRequireCalled ) {
+				throw 'bender.amd.require() must be called just once.';
+			}
+
+			wasRequireCalled = true;
+
+			var done = bender.defer();
+
+			var modules = [].slice.call( arguments );
+
+			CKEDITOR.require( modules, function() {
+				// Save all returned modules.
+				requires = [].slice.call( arguments );
+
+				// Call all describe()s that where waiting for `requires` to load.
+				flushDescribeQueue();
+
+				// Finally give green light for tests to start.
+				done();
+			} );
+		}
+	};
+
+	var originalDescribe = describe;
+	var describeQueue = [];
+
+	// Override the original Mocha's describe() so we can pass required modules to it.
+	describe = function( title, fn ) {
```

Comment:
Hum... no good news. `describe()` is supposed to return a "suite" object. We don't need it in our tests, but mocha needs it internally for `describe.only()`. As we're overriding `describe()` to delay its call, we cannot return anything and things break.

As a matter of fact, we don't need `describe.only()` as bender makes it possible to click in the test to run it only. So we could leave `only()` undefined and we would not suffer because of it.

I was able to make `describe.skip()` work well though. 

So we have two options:

 1. Go ahead with supporting `describe.skip()` with the current approach and keep `describe.done()` undefined.
 2. Change the whole thing and switch to the `var modules = bender.amd.require()` approach.

I would still go with "1" because it is cleaner, but this time I'll leave this to be decided by you guys.